src/qt/ScopeGuard.h

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) 2014-2020, The Monero Project
```

Comment:
We are not the authors of original code from Qt

Reply:
done

Reply:
Hm. I still think we shouldn't put Monero copyright here.